src/CsvReader.php

```diff
@@ -148,7 +129,7 @@ protected function readLine()
         // allow empty enclosure hack
         $enclosure = !$this->getEnclosure() ? chr(0) : $this->getEnclosure();
         $escapedBy = !$this->getEscapedBy() ? chr(0) : $this->getEscapedBy();
-        return fgetcsv($this->getFilePointer(), null, $this->getDelimiter(), $enclosure, $escapedBy);
+        return @fgetcsv($this->getFilePointer(), null, $this->getDelimiter(), $enclosure, $escapedBy);
```

Comment:
V PHP 7.4 (pridane do CI) funkcie `fgetcsv` a `fread` vyhodia chybu - ak invalid file pointer.
V starsich verziach sa vrati iba `null` alebo `false`, chyba sa tam nevyhodi.

Z pohladu uzivatela sa vrati prazdne pole, testuje sa to tu (tento test padal):
https://github.com/keboola/php-csv/blob/86f24a07ab30b7ed95b0991af735b8f6e3a0cd4b/tests/CsvReadTest.php#L477-L484

V `CsvWriter` si naopak generujeme chybovu spravu sami, tam to zostalo bez zmeny, a testuje sa to tu:
https://github.com/keboola/php-csv/blob/86f24a07ab30b7ed95b0991af735b8f6e3a0cd4b/tests/CsvWriteTest.php#L190-L215